library/Database.re

```diff
@@ -2,6 +2,8 @@ open Ezpostgresql;
 
 exception Database_not_configured;
 
+let wrapColumnValue = value => "'" ++ value ++ "'";
```

Comment:
isn't this more like wrapStringValue?

Reply:
probably we can make like this `encodeValue(String(foo)), encodeValue(Date(foo)) etc`

Reply:
hmm indeed, I was thinking only on strings but indeed, int, float and boolean don't need this